test/loader.test.js

```diff
@@ -249,15 +249,15 @@ describe("loader", () => {
     const testId = "./shallow-deep-literal.styl";
     const compiler = getCompiler(testId, {
       stylusOptions: {
-        dest: "deep/deep-literal.css",
+        dest: path.resolve(__dirname, "fixtures/"),
```

Comment:
Side effect after fixing logic, because developer can pass own options for source maps, we need to allow to change it too